src/RoomsSDKAdapter.integration.test.js

```diff
@@ -0,0 +1,46 @@
+import {removeTestUser} from '@webex/test-users';
+
+import {createIntegrationTestUser, createIntegrationSdkInstance} from './testHelper';
+import RoomsSDKAdapter from './RoomsSDKAdapter';
+
+describe('Rooms SDK Adapter', () => {
+  let roomsSDKAdapter;
+  let sdk;
+  let user;
+  let createdRoom;
+
+  beforeAll(async () => {
+    // Setting timeout higher for user creation and room creation
+    jest.setTimeout(15000);
+    user = await createIntegrationTestUser();
+    sdk = createIntegrationSdkInstance(user.token);
+    createdRoom = await sdk.rooms.create({title: 'Webex Teams Test Room'});
+  });
+
+  beforeEach(() => {
+    roomsSDKAdapter = new RoomsSDKAdapter(sdk);
+  });
+
+  describe('getRoom() returns', () => {
+    test('a room in a proper shape', (done) => {
+      roomsSDKAdapter.getRoom(createdRoom.id).subscribe((room) => {
+        expect(room).toMatchObject({
+          ID: createdRoom.id,
+          title: createdRoom.title,
+        });
+        done();
+      });
+    });
+  });
+
+  afterEach(() => {
+    roomsSDKAdapter = null;
+  });
+
+  afterAll(() =>
+    removeTestUser(user).catch((reason) => {
```

Comment:
should we use `async/await` / `try/catch` syntax for consistency?
```suggestion
try {
  await removeTestUser(user)
} catch(reason) {
```